docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -65,9 +65,58 @@ boosted tree regression model which combines multiple weak models into a
 composite one. We use decision trees to learn to predict the probability that 
 a data point belongs to a certain class.
 
+
+[discrete]
+[[dfa-classification-interpret]]
+==== Interpreting {classification} results
+
+The following sections help you understand and interpret the results of a 
+{classanalysis}.
+
+[discrete]
+[[dfa-classification-class-probability]]
+===== `class_probability`
```

Comment:
This information already appears in the example (flightdata-classification.html#flightdata-classification-results) and in my opinion that's a better place for it, unless we end up creating a separate page that dives into how to interpret classification results.

Reply:
@lcawl 
I think it is a good solution to collect this kind of info and publish it in one place.  You are right that this info appears on the example page and it is needed there for the full context. I also would like to add here, so you'll have a basic understanding of how to interpret results even if you don't read the examples. WDYT?